Nos/Views/Onboarding/OnboardingLoginView.swift

```diff
@@ -10,7 +10,7 @@ import Dependencies
 import Logger
 
 struct OnboardingLoginView: View {
-    var completion: () -> Void
+    let completion: @MainActor () -> Void
```

Comment:
Oh I didn't know you could annotate a closure type this way. Cool.

Reply:
Yeah, just tried and it worked. It was the only chance we had to not lose the mainactor in the onboarding completions